### PR TITLE
Git+Bash Assignment - Shree [WIP]

### PR DESCRIPTION
**Pull request by ShreeSinghi**

### CODE REVIEW

**Comment by divo12** [comment]
Nice progress shree!
An improvement what you could do is to document your work so that if someone wants to view, he gets an idea instantly

**Comment by ShreeSinghi** [comment]
Completed all tasks

**Comment by divo12** [comment]
@ShreeSinghi for the problem 4, write a script that basically on running takes your input n and degree of polynomial regression and runs the python file automatically. Also you can remove the git tutorial folder.
Nice work!

**Comment by guptaaryan16** [comment]
@ShreeSinghi done enough for the assignment. You can include some more features in your bash script like creating default values in your bash prompts and adding a path for storing the values if asked.

@divo12 check and merge please

**Comment by divo12** [comment]
@ShreeSinghi change the directory structure as told

**Comment by ShreeSinghi** [comment]
@divo12 Done

File: Git-Bash/hello.txt
Comment (guptaaryan16) [inline Git-Bash/hello.txt:1]: Remove this file

**Comment by ShreeSinghi** [comment]
Done

**Pull request merged**